waku/v2/protocol/waku_store/waku_store.nim

```diff
@@ -126,22 +126,21 @@ proc init*(T: type HistoryQuery, buffer: seq[byte]): ProtoResult[T] =
 
   # var topics: seq[ContentTopic]
 
-  # discard ? pb.getRepeatedField(2, topics)
-  # msg.topics = topics
   var buffs: seq[seq[byte]]
   discard ? pb.getRepeatedField(2, buffs)
   
   for buf in buffs:
     msg.contentFilters.add(? HistoryContentFilter.init(buf))
 
+  discard ? pb.getField(3, msg.pubsubTopic)
 
   var pagingInfoBuffer: seq[byte]
-  discard ? pb.getField(3, pagingInfoBuffer)
+  discard ? pb.getField(4, pagingInfoBuffer)
 
   msg.pagingInfo = ? PagingInfo.init(pagingInfoBuffer)
 
-  discard ? pb.getField(4, msg.startTime)
-  discard ? pb.getField(5, msg.endTime)
+  discard ? pb.getField(5, msg.startTime)
```

Comment:
Probably fine here since we are developing protocol but in general we should be careful about field other and changing it, since this breaks implementations. Generally it is preferred to append field numbers, as this means existing implementations works like normal.